projects/plugins/jetpack/_inc/lib/admin-pages/class-jetpack-search-dashboard-page.php

```diff
@@ -122,6 +126,11 @@ public function load_admin_scripts() {
 			$script_dependencies = $asset_manifest['dependencies'];
 		}
 
+		if ( ! ( new Status() )->is_offline_mode() && Jetpack::is_connection_ready() ) {
+			// Required for Analytics.
+			wp_enqueue_script( 'jp-tracks', '//stats.wp.com/w.js', array(), gmdate( 'YW' ), true );
```

Comment:
We could use `Automattic\Jetpack\Tracking::register_tracks_functions_scripts( true );` here instead.

Reply:
Good to know, thanks Jason
Changed in d1dc976203